File: twister-lib-android/capture_image/src/main/java/net/twisterrob/android/activity/FileUriExposedException.java
package net.twisterrob.android.activity;

/**
 * @see android.os.FileUriExposedException
 */
class FileUriExposedException extends RuntimeException {
	private static final long serialVersionUID = -1789299731318366566L;

	public FileUriExposedException(String message) {
		super(message);
	}
}

File: twister-lib-android/capture_image/src/main/java/net/twisterrob/android/activity/FileUriExposedException.kt
package net.twisterrob.android.activity

/**
 * @see android.os.FileUriExposedException
 */
internal class FileUriExposedException(message: String) : RuntimeException(message) {
	companion object {
		private const val serialVersionUID = -1789299731318366566L
	}
}
